src/transformStateWithClones.js

```diff
@@ -6,8 +6,44 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
   // write code here
+  const array = [];
+  const stateCopy = { ...state };
+  let pushedCopy = {};
```

Comment:
`let pushedCopy `is unnecessary. You could perform all operations in `let stateCopy`